src/Tools/DynamoFeatureFlags/FeatureFlagsClient.cs

```diff
@@ -87,7 +87,9 @@ internal FeatureFlagsClient(string userkey, string mobileKey = null, bool testMo
                     //in tests we want instancing on so we can test it.
                     { "graphics-primitive-instancing", LdValue.Of(true) },
                     //in tests we want search debouncing on so we can test it.
-                    { "searchbar_debounce", LdValue.Of(true) } });
+                    { "searchbar_debounce", LdValue.Of(true) },
```

Comment:
Do we still need this?

Reply:
ff `searchbar_debounce` is still needed. I needs to be validated before we unleash it